tests/system/filesystem_integrity/conftest.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
```

Comment:
We have to move the `filesystem_integrity` from system tests to outside.

Reply:
Solved https://github.com/wazuh/wazuh-qa/pull/2437/commits/7955e10644d7e0d5c4e2db1e82d28b85e367946a